renovate.json

```diff
@@ -21,8 +21,7 @@
     {
       "matchManagers": ["poetry"],
       "matchPackageNames": ["python"],
-      "preserveSemverRanges": true,
```

Comment:
This is intentional, right? Making sure since previous PR just added it

Reply:
Roger: turns out this was an invalid label. This PR's change does what we want.